gsa/src/web/pages/tasks/detailspage.js

```diff
@@ -383,6 +381,14 @@ const Page = ({
     }
   }, [history]);
 
+  const [loadOverrides, {overrides}] = useLazyGetOverrides({
+    filterString: 'task_id:' + id,
+  });
+
+  useEffect(() => {
+    loadOverrides();
+  }, [task]);
```

Comment:
Shouldn't it be `id` instead of `task`? Don't forget the task object changes with every reload. That means with every reload the overrides would be loaded again too (not considering apollo caching atm)
```suggestion
  }, [id]);
```